app/jobs/repository_audit_job.rb

```diff
@@ -0,0 +1,9 @@
+class RepositoryAuditJob < ApplicationJob
+  def perform(user: User.system_user)
+    packages = Package.stored
+    audit = Audit.create(user: user, packages: packages.count)
+    packages.each do |package|
+      AuditFixityCheckJob.perform_later(package: package, user: user, audit: audit)
+    end
+  end
```

Comment:
Let's try to return the audit instance here and see if a caller to `peform_now` receives it.